### PR TITLE
bugfix/parse-missing-wavelength

### PR DESCRIPTION
**Pull request by BrianWhitneyAI**
### Description 

This PR is a response to https://github.com/bioio-devs/bioio-czi/issues/19#issuecomment-2540320107. Our current transform from the czi schema to the OME schema does not account for an empty wavelength leading to parsing issues. Additionally, light sources need to be prefaced with "LightSource:"

### CODE REVIEW

File: xslt/Instrument.xsl [hunk 1]
```diff
@@ -95,17 +95,22 @@ ome/ome.xsd: 979 # # This means that for more details on how this section of the
...
             <xsl:choose>
-                <xsl:when test="LightSourceType/Laser">
-                    <xsl:element name="ome:Laser">
-                        <xsl:attribute name="ID">
-                            <xsl:value-of select="@Id"/>
-                        </xsl:attribute>
+            <xsl:when test="LightSourceType/Laser">
+                <xsl:element name="ome:Laser">
+                    <xsl:attribute name="ID">
+                        <xsl:text>LightSource:</xsl:text>
```
Comment (tlambert03) [inline xslt/Instrument.xsl]: fwiw ... this general pattern applies to *all* of the ID fields in the schema.  See the respective fields in the OME schema and you'll see the rules for the respective fields

Comment (toloudis) [reply xslt/Instrument.xsl]: @BrianWhitneyAI want to open a separate ticket to review our naming/usage to make sure this is satisfied everywhere?

**Review by toloudis — APPROVED** [review]
Not sure if indenting got messed up between line 97 and 98 but the content looks reasonable to me.

**Comment by toloudis** [comment]
Oh also maybe a unit test for this would be interesting.

File: xslt/Instrument.xsl (hunk 1 above)
Comment (BrianWhitneyAI) [reply xslt/Instrument.xsl]: #23

**Review by toloudis — APPROVED** [review]

**Pull request merged**